lib/node_modules/@stdlib/lapack/base/dlascl/README.md

```diff
@@ -0,0 +1,249 @@
+
+Note that indexing is relative to the first index. To introduce an offset, use [`typed array`][mdn-typed-array] views.
+
+<!-- eslint-disable stdlib/capitalized-comments -->
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+// Initial arrays...
+var A0 = new Float64Array( [ 0.0, 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
+
+// Create offset views...
+var A1 = new Float64Array( A0.buffer, A0.BYTES_PER_ELEMENT*1 ); // start at 2nd element
+
+dlascl( 'row-major', 'general', 0, 0, 1.0, 2.0, 3, 2, A1, 2 );
+// A0 => <Float64Array>[ 0.0, 2.0, 4.0, 6.0, 8.0, 10.0, 12.0 ]
+```
+
+#### dlascl.ndarray( type, KL, KU, CFROM, CTO, M, N, A, strideA1, strideA2, offsetA )
+
+Multiplies a real M by N matrix `A` by a real scalar `CTO/CFROM` using alternative indexing semantics.
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var A = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] ); // => [ [ 1.0, 2.0 ], [ 3.0, 4.0 ], [ 5.0, 6.0 ] ]
+
+dlascl.ndarray( 'general', 0, 0, 1.0, 2.0, 3, 2, A, 2, 1, 0 );
+// A => <Float64Array>[ 2.0, 4.0, 6.0, 8.0, 10.0, 12.0 ]
+```
+
+The function has the following additional parameters:
+
+-   **type**: specifies the type of matrix `A` ( should be one of these: `general`, `upper`, `lower`, `upper-hessenberg`, `symmetric-banded-lower`, `symmetric-banded-upper` or `banded` ).
+-   **KL**: lower band width of `A`. Referenced only if type is `symmetric-banded-lower` or `banded`.
+-   **KU**: upper band width of `A`. Referenced only if type is `symmetric-banded-upper` or `banded`.
+-   **CFROM**: the matrix `A` is multiplied by `CTO / CFROM`.
+-   **CTO**: the matrix `A` is multiplied by `CTO / CFROM`.
+-   **M**: number of rows in matrix `A`.
+-   **N**: number of columns in matrix `A`.
+-   **A**: input [`Float64Array`][mdn-float64array].
+-   **strideA1**: stride of the first dimension of `A`.
+-   **strideA2**: stride of the second dimension of `A`.
+-   **offsetA**: starting index for `A`.
+
+If `type` is `banded`, `symmetric-banded-lower` or `symmetric-banded-upper` the matrix should be stored in the [`Band storage`][lapack-band-storage] format.
+
+While [`typed array`][mdn-typed-array] views mandate a view offset based on the underlying buffer, the offset parameters support indexing semantics based on starting indices. For example,
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var A = new Float64Array( [ 0.0, 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
+
+dlascl.ndarray( 'general', 0, 0, 1.0, 2.0, 3, 2, A, 2, 1, 1 );
+// A => <Float64Array>[ 0.0, 2.0, 4.0, 6.0, 8.0, 10.0, 12.0 ]
+```
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+## Notes
+
+-   `dlascl()` corresponds to the [LAPACK][lapack] routine [`dlascl`][lapack-dlascl].
+-   If `type` is `banded`, `symmetric-banded-lower` or `symmetric-banded-upper` the matrix should be stored in the [`Band storage`][lapack-band-storage] format.
```

Comment:
```suggestion
-   If `type` is `'banded'`, `'symmetric-banded-lower'`, or `'symmetric-banded-upper'`, the matrix should be stored in the [band storage][lapack-band-storage] format.
```

Reply:
May be good to provide a brief schematic using math equations in terms of what is meant.